src/js/components/sharedComponents/header/searchBar/SearchBar.jsx

```diff
@@ -26,7 +26,7 @@ export default class SearchBar extends React.Component {
 
     expandSearch() {
         this.setState({
-            expanded: false
+            expanded: true
```

Comment:
does this change the ui's behavior is anyway?  i tested and i didn't see any change so just asking if this could cause any unexpected results eventually?

Reply:
it looks like this instance of searchbar might not even be used anywhere, i searched for 'SearchBar' and all the other components i found seem to be using the one from dtui